BitwardenShared/UI/Vault/Vault/AutofillList/VaultAutofillListState.swift

```diff
@@ -43,4 +46,9 @@ struct VaultAutofillListState: Equatable, Sendable {
 
     /// The list of sections to display for matching vault items.
     var vaultListSections = [VaultListSection]()
+
+    /// Whether to show the add item button.
+    var showAddItemButton: Bool {
+        !isAutofillingFido2List && !isAutofillingTotpList
```

Comment:
❓ Do we want to hide the add item button in the navigation bar when `isAutofillingFido2List`?

Related conversation: https://github.com/bitwarden/ios/pull/884#discussion_r1833162574

Reply:
Nice catch, fixed! Sorry I think I did some of the work while working on the Beta and the change of Fido2 showing the button was made afterwards and that's why I have some of that stuff mixed up. Thanks again for catching all of this and sorry for the inconvenience!

Reply:
👍 No problem!